### PR TITLE
feat: setup github actions

### PR DESCRIPTION
**Pull request by GuillaumeBarra**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    strategy:
+      matrix:
+        node_version: [18]
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node_version }}
+        uses: actions/setup-node@v3
+        with:
```
Comment (CallumHemsley) [inline .github/workflows/ci.yml:18]: Should be able to do ```with:
    node-version: 16```

File: .eslintrc.js [hunk 2]
```diff
@@ -29,8 +29,5 @@ module.exports = {
   rules: {
     "import/order": "off",
     "react-hooks/exhaustive-deps": "error",
-    "prettier/prettier": ["warn", {
```
Comment (juskek) [inline .eslintrc.js:32]: why was this removed? this will require a change in indentation for all the files in the code base right? 😰

**Review by justinkek — APPROVED** [review]

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "printWidth": 120,
+    "tabWidth": 4,
```
[the remark this answers is not in the document]
Comment (Francisco-Costa98) [reply .prettierrc:3]: so the prettierrc is the file that most vscodes are setup to read from when you have pritier installed 

so i moved this here so my format on save could work properly and itll still match the linting
